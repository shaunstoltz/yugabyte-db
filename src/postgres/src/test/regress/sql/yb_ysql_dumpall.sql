CREATE TABLESPACE x LOCATION '/data';
CREATE TABLESPACE y WITH (replica_placement='{"num_replicas":3, "placement_blocks":[{"cloud":"cloud1","region":"r1","zone":"z1","min_num_replicas":1},{"cloud":"cloud2","region":"r2", "zone":"z2", "min_num_replicas":2}]}');
CREATE TABLESPACE z WITH (replica_placement='{"num_replicas":1, "placement_blocks":[{"cloud":"cloud1","region":"datacenter1","zone":"rack1","min_num_replicas":1}]}');

CREATE TABLE table1(id int) TABLESPACE x;
CREATE INDEX idx1 on table1(id) TABLESPACE z;
CREATE TABLE table2(name varchar) TABLESPACE z;
CREATE INDEX idx2 on table2(name) TABLESPACE x;

DROP TABLESPACE y;
