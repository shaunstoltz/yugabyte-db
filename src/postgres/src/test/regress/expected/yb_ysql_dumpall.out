--
-- YSQL database cluster dump
--

SET default_transaction_read_only = off;

SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;

--
-- Roles
--

CREATE ROLE postgres;
ALTER ROLE postgres WITH SUPERUSER INHERIT CREATEROLE CREATEDB LOGIN REPLICATION BYPASSRLS;
CREATE ROLE yugabyte;
ALTER ROLE yugabyte WITH SUPERUSER INHERIT CREATEROLE CREATEDB LOGIN REPLICATION BYPASSRLS PASSWORD 'md52c2dc7d65d3e364f08b8addff5a54bf5';
CREATE ROLE yugabyte_test;
ALTER ROLE yugabyte_test WITH SUPERUSER INHERIT CREATEROLE CREATEDB LOGIN NOREPLICATION BYPASSRLS;




--
-- Tablespaces
--

CREATE TABLESPACE x OWNER yugabyte_test LOCATION '';
CREATE TABLESPACE z OWNER yugabyte_test LOCATION '' WITH (replica_placement='{"num_replicas":1, "placement_blocks":[{"cloud":"cloud1","region":"datacenter1","zone":"rack1","min_num_replicas":1}]}');


\connect template1

--
-- YSQL database dump
--

-- Dumped from database version 11.2-YB-2.5.1.0-b0
-- Dumped by ysql_dump version 11.2-YB-2.5.1.0-b0

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- YSQL database dump complete
--

\connect postgres

--
-- YSQL database dump
--

-- Dumped from database version 11.2-YB-2.5.1.0-b0
-- Dumped by ysql_dump version 11.2-YB-2.5.1.0-b0

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- YSQL database dump complete
--

--
-- YSQL database dump
--

-- Dumped from database version 11.2-YB-2.5.1.0-b0
-- Dumped by ysql_dump version 11.2-YB-2.5.1.0-b0

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- Name: system_platform; Type: DATABASE; Schema: -; Owner: postgres
--

CREATE DATABASE system_platform WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'C' LC_CTYPE = 'en_US.UTF-8';


ALTER DATABASE system_platform OWNER TO postgres;

\connect system_platform

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- Name: DATABASE system_platform; Type: COMMENT; Schema: -; Owner: postgres
--

COMMENT ON DATABASE system_platform IS 'system database for YugaByte platform';


--
-- YSQL database dump complete
--

--
-- YSQL database dump
--

-- Dumped from database version 11.2-YB-2.5.1.0-b0
-- Dumped by ysql_dump version 11.2-YB-2.5.1.0-b0

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- Name: yugabyte; Type: DATABASE; Schema: -; Owner: postgres
--

CREATE DATABASE yugabyte WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'C' LC_CTYPE = 'en_US.UTF-8';


ALTER DATABASE yugabyte OWNER TO postgres;

\connect yugabyte

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- Name: DATABASE yugabyte; Type: COMMENT; Schema: -; Owner: postgres
--

COMMENT ON DATABASE yugabyte IS 'default administrative connection database';


SET default_tablespace = x;

SET default_with_oids = false;

--
-- Name: table1; Type: TABLE; Schema: public; Owner: yugabyte_test; Tablespace: x
--

CREATE TABLE public.table1 (
    id integer
);


ALTER TABLE public.table1 OWNER TO yugabyte_test;

SET default_tablespace = z;

--
-- Name: table2; Type: TABLE; Schema: public; Owner: yugabyte_test; Tablespace: z
--

CREATE TABLE public.table2 (
    name character varying
);


ALTER TABLE public.table2 OWNER TO yugabyte_test;

--
-- Data for Name: table1; Type: TABLE DATA; Schema: public; Owner: yugabyte_test
--

COPY public.table1 (id) FROM stdin;
\.


--
-- Data for Name: table2; Type: TABLE DATA; Schema: public; Owner: yugabyte_test
--

COPY public.table2 (name) FROM stdin;
\.


--
-- Name: idx1; Type: INDEX; Schema: public; Owner: yugabyte_test; Tablespace: z
--

CREATE INDEX idx1 ON public.table1 USING lsm (id HASH);


SET default_tablespace = x;

--
-- Name: idx2; Type: INDEX; Schema: public; Owner: yugabyte_test; Tablespace: x
--

CREATE INDEX idx2 ON public.table2 USING lsm (name HASH);


--
-- YSQL database dump complete
--

--
-- YSQL database cluster dump complete
--

